aldryn_config.py

```diff
@@ -17,12 +18,29 @@ def clean(self, value):
         return u", ".join(class_names)
 
 
+class TagTypesField(forms.CharField):
+
+    def clean(self, value):
+        value = super(TagTypesField, self).clean(value)
+        tag_types = filter(bool, map(lambda x: x.strip(), value.split(',')))
+        for tag_type in tag_types:
+            if not TAG_TYPE_FORMAT.match(tag_type):
+                raise forms.ValidationError(
+                    u'%s does not look like a proper HTML tag.' % (tag_type, ))
+        return u", ".join(tag_type)
```

Comment:
i'd prefer single quotes and new style string formatting (https://pyformat.info/)